lab_7_llm/main.py

```diff
@@ -136,6 +168,20 @@ def analyze_model(self) -> dict:
         Returns:
             dict: Properties of a model
         """
+        tensor_data = torch.ones(1, 512, dtype=torch.long)
```

Comment:
512 - "magic number". Take this parameter from `model.config`

Reply:
done